CHANGELOG.md

```diff
@@ -51,7 +65,6 @@ This release adds support for running CTL contracts against Babbage-era nodes. *
 - Modules `Metadata.Seabug` and `Metadata.Seabug.Share`
 - `POST /eval-ex-units` Haskell server endpoint ([#665](https://github.com/Plutonomicon/cardano-transaction-lib/pull/665))
 - Truncated test fixtures for time/slots inside `AffInterface` to test time/slots not too far into the future which can be problematic during hardforks https://github.com/Plutonomicon/cardano-transaction-lib/pull/676
-- `d` and `extraEntropy` protocol parameters from protocol parameters update proposal
```

Comment:
Why is this line removed? Bad merge?

Reply:
@klntsky Probably yes, I also noticed there're some issues after merge, fixing it now

Reply:
@klntsky It looks like that change is in the wrong section anyway, it came after v2.0.0-alpha